Consul.Test/LockTest.cs

```diff
@@ -269,9 +269,7 @@ public async Task Lock_ContendWait()
                 }));
             }
 
-            await Task.WhenAny(
-                Task.WhenAll(tasks),
-                Task.Delay(TimeSpan.FromTicks(contenderPool * Lock.DefaultLockWaitTime.Ticks)));
+            await WithTimeout(Task.WhenAll(tasks), TimeSpan.FromMinutes(5));
```

Comment:
What is the rationale for waiting up to 5 minutes in this case, instead of 1 minute like in most other tests?